### DISCUSSION TITLE
Issue with column data

### DISCUSSION THREAD

**Discussion by paur94** [KevinVandy/material-react-table#578] [Q&A]
Hi there,

In the LoanFragment, there are missing value errors, but my table is working well. However, when I added a new "LoanId" value to the LoanFragment type, I encountered the issue "Type instantiation is excessively deep and possibly infinite." I haven't edited the table; I am only adding a new value to the LoanFragment.
![image](https://github.com/KevinVandy/material-react-table/assets/60259252/2184aa9d-5224-4b1b-94a5-e6f889f6629e)

**Comment by KevinVandy** [comment]
What version of Typescript are you on? TanStack Table requires at least v4.8 I think, but you should try to keep it as up to date as possible

**Reply by paur94** [reply]
5.1.3, the latest version of TS

**Reply by paur94** [reply]
Sorry, have you any updates?🙄

**Reply by KevinVandy** [reply]
Is your data type recursive? TanStack Table struggles with infinite recursive types.

**Reply by paur94** [reply]
No, I have never made changes to the table within that timeframe. I have only added a new value to LoanFragment and afterward, issues occurred.
